source/about/install-mattermost-for-microsoft-teams-plugin.rst

```diff
@@ -128,6 +132,58 @@ Configure the plugin
 
 Additional configuration settings are available for this plugin. See the `MS Teams Sync plugin configuration settings </configure/plugins-configuration-settings.html#ms-teams-sync>`__ documentation for details.
 
+Monitor plugin performance
+--------------------------
+
+You can set up `performance monitoring </scale/performance-monitoring.html>`__ and `performance alerting </scale/performance-alerting.html>`__ for this plugin using Prometheus and Grafana.
+
+- Monitoring enables you to proactively review the overall health of the plugin, including database calls, HTTP requests, and API latency.
+- Alerting enables you to detect and take action as issues come up, such as the integration being offline.
+
+Grafana dashboards `are available on GitHub <https://github.com/mattermost/mattermost-plugin-msteams-sync/blob/main/server/metrics/dashboards/cloud.json>`__ for Mattermost Cloud deployments as a useful starting point. These dashboards are designed for use in Mattermost Cloud, and filter to a given ``namespace``. 
+
+.. image:: ../images/grafana-dashboard-msteams.png
+  :alt: Example of a Grafana monitoring dashboard for a Mattermost instance connected to Microsoft Teams.
+
+.. note:: 
+  
+  Modifications will be necessary for self-hosted Mattermost deployments. See the `Get help <#get-help>`__ section below for details on how to contact us for assistance.
+
+Respond to issue alerts
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Once performance alerts are set up, if you receive an performance issue alert, we recommend completing the following troubleshooting and debugging steps before reaching out to Mattermost for assistance:
+
+1. If you have confirmed the integration isn't working for an individual user, but works for others, ask the user experiencing the issue to disconnect from the MS Teams integration using ``/msteams-sync disconnect``, then re-connect using ``/msteams-sync connect``.
+2. If you have confirmed the integration isn't working for several users, go to **System Console > Plugin Management**, search for **MSTeams Sync**, disable the plugin, and then & re-enable it.
```

Comment:
```suggestion
2. If you have confirmed the integration isn't working for several users, go to **System Console > Plugin Management**, search for **MSTeams Sync**, disable the plugin, and then re-enable it.
```

I understand the history of where this came from -- the plugin crashing and going into a failure state -- but I'm inclined not to suggest this as a general solution. It reads a bit like "Rebooting Windows and wave hands in the air," and we're intending to fix the underlying cause here to prevent this from ever happening again. Maybe we just omit this step?

Reply:
@jasonblais - Thoughts?